mono/utils/mono-error-internals.h

```diff
@@ -144,9 +144,6 @@ MonoException*
 mono_error_convert_to_exception (MonoError *error);
 
 void
-mono_error_raise_exception (MonoError *error);
```

Comment:
this was moved to metadata with  https://github.com/mono/mono/commit/b80d06dfa0046b2d13ee99508c4162a0edfb9091 so I think it should be removed here right?